Makefile

```diff
@@ -28,7 +28,7 @@ test: ## Run unit tests
 	mkdir -p ${TEST_DIR}
 	cp .coveragerc ${TEST_DIR}
 	cp setup.cfg ${TEST_DIR}
-	cd ${TEST_DIR}; python -m pytest
```

Comment:
what about the other `pyargs` in this file? Also there are more flags, we could remove them as well. I might have forgotten them last time when I consolidated them or someone has added them recently?

Reply:
I don't know - tbh, I was just hacking around semi-blindly until it worked.

Can you explain to me what specifically you mean with "what about"?

If you mean the `nosoftdeps` test, we can leave `pyargs` in there since it runs only a small number of specific tests which we probably always want, they take only a few minutes.